### PR TITLE
[KYC] Update KYC verification documentation

### PR DESCRIPTION
**Pull request by havan**
User guide for KYC Verification is updated with revised text and new and detailed screenshots.

Deployed at: https://playground.docs.camino.network/guides/kyc

### CODE REVIEW

File: docs/guides/kyc.md [hunk 1]
```diff
@@ -7,62 +7,177 @@ title: KYC Verification
...
+the identity and other credentials of a user. It is legally mandatory in most countries
+to ensure users are who they claim to be and to help against money laundering and
+the financing of terrorist activities. Additionally, if users on the Camino network are
+verified, it enhances trustworthiness and reduces risk, thereby improving the overall
+experience.
 
-To be verified, the user has to share an id document and pass a liveness check to authenticate. After a successful verification, we save the verified status to the Camino blockchain where it can be picked up by the network.
+To be verified, the user must share an ID document and pass a liveness check to authenticate.
+After successful verification, the verified status is saved to the Camino blockchain where
+it can be picked up by the network.
```
Comment (xdaluca) [inline docs/guides/kyc.md]: Would change to:
After a successful verification, the verified KYC status is associated with the user wallet. It allows other network participants and builders to know which wallets have a verified KYC, without sharing any of the personal KYC information that the user provided.

**Review by xdaluca — APPROVED** [review]
Added a small suggestion for 1 sentence, aside from that LGTM.

File: docs/guides/kyc.md (hunk 1 above)
Comment (havan) [reply docs/guides/kyc.md]: Thanks, updated.

**Review by xdaluca — APPROVED** [review]
LGTM

**Pull request merged**